test-tests/shared/src/test/scala/zio/test/GenSpec.scala

```diff
@@ -1,14 +1,12 @@
 package zio.test
 
-import zio.duration.{Duration, _}
-import zio.random.Random
+import zio._
 import zio.test.Assertion._
 import zio.test.GenUtils._
 import zio.test.TestAspect.{nonFlaky, scala2Only, setSeed}
 import zio.test.{check => Check, checkN => CheckN}
-import zio.{Chunk, NonEmptyChunk, ZIO}
 
-import java.time._
+import java.time.{Duration => _, _}
```

Comment:
This is actually the bigger annoyance than name clashes with `scala.concurrent.Duration`.

Reply:
Yeah :(